catalog/pipeline/deploy-release/0.4/deploy-release.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: deploy-release
+  labels:
+    app.kubernetes.io/version: "0.4"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton pipeline to verify Snapshot prior to Deployment
+  params:
+    - name: release
+      type: string
+      description:
+        The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution
+    - name: releaseplan
+      type: string
+      description: The namespaced name (namespace/name) of the releasePlan
+    - name: releaseplanadmission
```

Comment:
I wonder if we should use camelCase here. It's not a big deal. But we already got `enterpriseContractPolicy`, so maybe it would make sense to continue with this? (This applies to `releasestrategy` and `releaseplan` as well.)

Reply:
The `WithObjectReferences` function in the operator changes it all to lowercase 
https://github.com/redhat-appstudio/release-service/blob/main/tekton/pipeline_run.go#L118
I actually failed on this which is why I changed it away from camelCase. We don't pass the EnterpriseContractPolicy using `WithObjectReferences` right now, which is why it is okay in camelCase. It will probably change in the near future though

Reply:
I see, thanks for the explanation.